Liroy/bootstrap.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
+
+#Step 1 - Update CentOS 7 system
+echo "Updating system"
+sudo yum -y install epel-release
```

Comment:
Let's assume the script is been invoked by root. Is sudo still required?
If not, is there any other way to invoke the script with sudo without specifying it on (almost) every command?